### DISCUSSION TITLE
Recuperare tutte le fatture scadute

### DISCUSSION THREAD

**Discussion by ghost** [fattureincloud/api#323] [Q&A] [answered]
All'interno di FattureInCloud,  nella lista fatture, esiste il tasto/filtro fatture scadute, ci servirebbe avere lo stesso da API per poter avere la lista di quelle fatture esattamente come abbiamo da dentro l'applicazione. Se qualcuno ha un esempio di codice (in qualsiasi linguaggio va bene) ringraziamo anticipatamente.

Matteo Anselmi
www.cyber3k.it

**Comment by emajo** [comment, marked as answer]
Ciao @cyber3k-srl per estrarre solo le fatture scadute è necessario fare una chiamata [List Issued Documents](https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/issued_documents) mettendo il query parameter `q = next_due_date < '2023-05-18'`.

Grazie,
Emanuele.

**Reply by valmoz** [reply]
Buongiorno @cyber3k-srl 
abbiamo risolto i suoi dubbi?

Grazie
Mauro

**Comment by ghost** [comment]
Buongiorno,
si la vostra risposta è stata risolutiva.

Grazie e buona giornata,


Matteo Anselmi

**Reply by valmoz** [reply]
Perfetto, grazie mille.

Mauro
